.github/workflows/docker.yml

```
@@ -0,0 +1,139 @@
+name: Docker unstable release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  ssr:
+    name: Build SSR docker image
```

```suggestion
    name: Build SSR image
```

It's already mentioned in the workflow name that it's for Docker.